cdc/entry/mounter.go

```diff
@@ -434,6 +434,13 @@ func (m *mounter) verifyChecksum(
 		decoder = m.decoder
 	}
 
+	version := decoder.ChecksumVersion()
+	if version != 0 {
+		log.Info("checksum is not v1, this happens since upstream TiDB is in a higher version, ignore it",
```

Comment:
Would this log flooding?

Reply:
this only happens in the upgrade scenario, it should not take too long, so not too match logs.

Reply:
Rolling upgrade would take more than 10 minutes if there are a lot of tables.